TST: Windows job fails with unistd.h error

Last successful run on `master` (cron): https://travis-ci.org/github/astropy/astropy/builds/722439530
First seen on `master` (cron): https://travis-ci.org/github/astropy/astropy/builds/722669911

They both attempt to run with astropy at 679a74a46379482 because nothing new was merged between the two scheduled runs. Somehow, one was successful but one failed with `fatal error C1083: Cannot open include file: 'unistd.h': No such file or directory`.

A quick glance did not reveal any obvious upstream changes (like conda or MSVSC updates).

This is blocking unrelated PRs and the rest of the cron jobs.

--- a/.readthedocs.yml
+++ b/.readthedocs.yml
@@ -12,6 +12,7 @@ sphinx:
 # Then, install special pinning for RTD.
 python:
   version: 3.8
+  system_packages: false
   install:
     - method: pip
       path: .

--- a/astropy/io/fits/setup_package.py
+++ b/astropy/io/fits/setup_package.py
@@ -31,12 +31,12 @@ def _get_compression_extension():
             # which ensures on windows we do not include unistd.h (in regular
             # compilation of cfitsio, an empty file would be generated)
             cfg['extra_compile_args'].extend(
-                ['/D', '"WIN32"',
-                 '/D', '"_WINDOWS"',
-                 '/D', '"_MBCS"',
-                 '/D', '"_USRDLL"',
-                 '/D', '"_CRT_SECURE_NO_DEPRECATE"',
-                 '/D', '"FF_NO_UNISTD_H"'])
+                ['/D', 'WIN32',
+                 '/D', '_WINDOWS',
+                 '/D', '_MBCS',
+                 '/D', '_USRDLL',
+                 '/D', '_CRT_SECURE_NO_DEPRECATE',
+                 '/D', 'FF_NO_UNISTD_H'])
         else:
             cfg['extra_compile_args'].extend([
                 '-Wno-declaration-after-statement'

